src/js/tech/flash.js

```diff
@@ -268,6 +268,16 @@ class Flash extends Tech {
   }
 
   /**
+   * Get the last error object
+   *
+   * @return {Object}
+   * @method error
+   */
+  error() {
+    return this.error_ || {};
```

Comment:
> or a black object

Sounds ominous. :)

If possible we should return `null` for no error to match the video element.